File: .conf/.vim/vimfiles/plugin/.vimrc.py
" ---------------------------------------------------------------------------------------
"  .vimrc.py
"  @tasasaki
" ---------------------------------------------------------------------------------------

echo "Load python's settings."

" ---------------------------------------------------------------------------------------
" python
" ---------------------------------------------------------------------------------------

" python
autocmd FileType python setl autoindent
autocmd FileType python setl smartindent cinwords=if,elif,else,for,while,try,except,finally,def,class
autocmd FileType python setl tabstop=8 shiftwidth=4 softtabstop=4

File: .conf/.vim/vimfiles/plugin/.vimrc.rb
" ---------------------------------------------------------------------------------------
"  .vimrc.rb
"  @tasasaki
" ---------------------------------------------------------------------------------------

echo "Load ruby's settings."

" ---------------------------------------------------------------------------------------
" ruby
" ---------------------------------------------------------------------------------------

" ruby
autocmd FileType ruby,eruby,yaml set softtabstop=2 shiftwidth=2 tabstop=2
autocmd FileType ruby,eruby let g:rubycomplete_buffer_loading = 1
autocmd FileType ruby,eruby let g:rubycomplete_rails = 1
autocmd FileType ruby,eruby let g:rubycomplete_classes_in_global = 1
let g:rails_level = 4


